guide/_sidebar.md

```diff
@@ -34,6 +34,9 @@
 	* [Getting started](/sharding/)
 	* [Additional information](/sharding/additional-information)
 
+	* **Canvas**
+	* [Canvas](/canvas/)
```

Comment:
Actually curious: do you plan on making more guide pages for canvas-related things? Because if not, I think this could be moved under Popular Topics. But if you do plan on adding more content, no problem here.

However, a more descriptive title would attract more attention to the link.